resources/js/Pages/EventList.vue

```diff
@@ -0,0 +1,101 @@
+<script setup lang="ts">
+import AppHead from '@/Components/AppHead.vue'
+import Navbar from '@/Components/Navbar.vue'
+import BaseInput from '@/Components/BaseInput.vue'
+import InfoBlock from '@/Components/InfoBlock.vue'
+import EventCard from '@/Components/EventCard.vue'
+import BaseButton from '@/Components/BaseButton.vue'
+import { MapPinIcon, CalendarIcon } from '@heroicons/vue/24/outline'
+import { formatDate } from '@/utilities/formatDate'
+import { useEvents } from '@/composables/useEvents'
+import PaginationComponent from '@/Components/PaginationComponent.vue'
+
+const { events, search, page, meta } = useEvents()
+</script>
+
+<template>
+  <AppHead title="Wydarzenia" />
+  <div class="flex items-center flex-col w-full">
+    <Navbar />
+
+    <div class="md:w-5/6 flex flex-col">
+      <div class="md:mb-36 mb-16">
+        <div class="w-full relative">
+          <img src="/images/Fade.svg"
+               alt=""
+               class="absolute inset-0 w-full h-[1000px] top-[-430px] pointer-events-none"
```

Comment:
It's recommended to use `rem` units instead of `px` for better scalability and accessibility